diff --git a/autogen.sh b/autogen.sh
index 83367f32..9ef871d8 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -6,7 +6,7 @@
 # re-generate files needed by configure, and created otherwise at 'dist' time
 if [ ! -f scripts/augeas/nutupsconf.aug.in ]
 then
-	if python -c "import re,glob,codecs"; then
+	if python3 -c "import re,glob,codecs"; then
 		echo "Regenerating Augeas ups.conf lens..."
 		cd scripts/augeas && {
 			./gen-nutupsconf-aug.py || exit 1
diff --git a/scripts/augeas/gen-nutupsconf-aug.py b/scripts/augeas/gen-nutupsconf-aug.py
index c443f43a..fea46559 100755
--- a/scripts/augeas/gen-nutupsconf-aug.py
+++ b/scripts/augeas/gen-nutupsconf-aug.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #   Copyright (C) 2010 - Arnaud Quette <arnaud.quette@gmail.com>
 #
 #   This program is free software; you can redistribute it and/or modify
